test/SimpleImportTest/CrawlerProcessor/JobProcessorTest.php

```diff
@@ -107,23 +121,143 @@ protected function initTarget()
         $this->dataInputFilter = $this->getMockBuilder(InputFilterInterface::class)
             ->getMock();
 
+        $this->fs = $this->getMockBuilder(Filesystem::class)
+            ->getMock();
+
+        $this->lockDir = realpath(__DIR__.'/../../sandbox/var/simple-import');
+        if(!is_dir($this->lockDir)){
+            mkdir($this->lockDir,0777,true);
+        }
```

Comment:
This gives me errors, when I'm running the tests locally in my Development Environment.  
"mkdir: invalid path"